addons/website_event_questions/views/website_event_questions_templates.xml

```diff
@@ -6,15 +6,24 @@
         <!-- Attendee specific questions -->
         <xpath expr="//t[@name='attendee_loop']" position="inside">
             <t t-if="event.specific_question_ids">
-                <div class="row mb32">
-                    <div class="col-md-4" t-foreach='event.specific_question_ids' t-as='question'>
-                        <div class="input-group input-group-sm">
-                            <span class="input-group-addon"><t t-esc="question.title"/></span>
-                            <select t-attf-name="#{counter}-answer_ids-#{question.id}" class="form-control" required="true">
-                                <t t-foreach='question.answer_ids' t-as="answer">
-                                    <option t-esc='answer.name' t-att-value='answer.id'/>
-                                </t>
-                            </select>
+                <script>
+                    $(document).ready(function() {
```

Comment:
@hmo-odoo The reason behind writing script in template is, this template is loaded in popup, if I write external script and add it into assets then note that before this popup is opened, assets were already loaded, so I can not bind select2 on element which is still not exist in DOM, even if I write external script then I need to add that script in this template only(instead of asset).